control/veda_deliver_cielo.py

```diff
@@ -136,7 +136,7 @@ def perform_transcript(self, job_id, lang_code):
             api_token=self.api_key,
             priority=self.turnaround,
             transcription_fidelity=self.fidelity,
-            options=json.dumps({"return_iwp": ["FINAL"]})
+            options=json.dumps({"return_iwp":["FINAL"]})
```

Comment:
wasn't space better for quality ?

Reply:
it was causing issues during url encoding so we removed it